stacker/lookups/handlers/ssmstore.py

```diff
@@ -0,0 +1,50 @@
+import boto3
+
+from ...util import read_value_from_path
+
+TYPE_NAME = "ssmstore"
+
+
+def handler(value, **kwargs):
+    """Retrieve (and decrypt if applicable) a parameter from AWS SSM Parameter Store.
+
+    ssmstore field types should be in the following format:
+
+        [<region>@]ssmkey
+
+    Note: The region is optional, and defaults to us-east-1 if not given.
+
+    For example:
+
+        # In stacker we would reference the encrypted value like:
+        conf_key: ${ssmstore us-east-1@ssmkey}
+
+        You can optionally store the value in a file, ie:
+
+        ssmstore_value.txt
+        us-east-1@ssmkey
+
+        and reference it within stacker (NOTE: the path should be relative to
+        the stacker config file):
+
+        conf_key: ${ssmstore file://ssmstore_value.txt}
+
+        # Both of the above would resolve to
+        conf_key: PASSWORD
+
+    """
+    value = read_value_from_path(value)
+
+    region = "us-east-1"
+    if "@" in value:
+        region, value = value.split("@", 1)
+
+    client = boto3.client("ssm", region_name=region)
+    response = client.get_parameters(
+        Names=[
+            value,
+        ],
+        WithDecryption=True
+    )
+
+    return response['Parameters'][0]['Value']
```

Comment:
This will throw an IndexError if the parameter doesn't exist, right?  I wonder if there might be a better error we can return that'll give folks a better idea of where this is breaking?  Maybe a new exception type?